package-lock.json

```diff
@@ -1,9645 +1,8 @@
 {
   "name": "@asyncapi/website",
   "version": "0.1.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
   "requires": true,
-  "packages": {
```

Comment:
@starlightknown If you can, kindly remove these changes from PR because this is made because of the wrong node version used by you. Use node `v12.22.0` and npm `v6.14.11`. This will not change the package-lock.json file.

Reply:
Yep will do that. Thank you

Reply:
Nope, you don't have to delete that file. You need to remove the changes made on that file.